core/src/main/java/com/digitalpebble/stormcrawler/filtering/URLFilters.java

```diff
@@ -49,7 +56,7 @@ private URLFilters() {
         filters = new URLFilters[0];
     }
 
-    private String configFile = "urlfilters.config.file";
+    private String configFile = "urlfilters.json";
```

Comment:
Is there a reason for actually changing this value? Was it a bug? :)

Reply:
in a way yes, it was a confusion between the key name for loading from the conf with the placeholder actual value. Didn't matter much as the default value was always overridden but at least now it is correct and in a way documents what the default value is